### PR TITLE
Modified the documentation for scaling up/down the Data Center cluster

### PR DESCRIPTION
**Pull request by nghazali**
The documentation on README.rd is not clear in terms up how to scale up/down the data center after installation.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -106,15 +106,13 @@ See [CONFIG.md]() for examples of how to configure the volumes.
...
 
-It is important to note, however, that both Jira and Confluence must initially
-be deployed with a `replicaCount` of 1, then be fully configured via the web interface,
-and only once setup is complete can they be scaled up to more replicas.
-
-In the case of Bitbucket, this intermediate step is unecessary, and a BitBucket
-deployment can be configure with a >1 `replicaCount` from the start.
\ No newline at end of file
+Please note that `replicaCount` s used by the functest helm chart to create "backdoor" ingress routes to individual cluster nodes. 
+It doesn't affect the number of PODs deployment, it just creates the routes.
```
Comment (kennymacleod) [inline INSTALL.md:118]: Let's avoid mentioning the functest stuff in the docs, it just confused the issue. 

Anyway, the new text is incorrect. The functest chart does not consult the `replicaCount` when adding the backdoor routes, it uses `helm_parameters` in `src/test`.

**Review by kennymacleod — CHANGES_REQUESTED** [review]
Remove undesirable and incorrect reference to the functest chart,

File: INSTALL.md (hunk 1 above)
Comment (nghazali) [reply INSTALL.md]: Thanks Kenny, done.

**Review by nghazali — COMMENTED** [review]
Done

**Review by kennymacleod — APPROVED** [review]

**Pull request merged**